File: module/module.ts
const Converter = (()=>{
    const FOOT = 30.48
    const INCH = 2.54 
    const MILE = 1609.34

    return {
        inchesToCentimeters: (inches) => inches * INCH,
        feetToMeters: (feet) => feet * FOOT,
        milesToKilometers: (miles) => miles * MILE
    }

})()

export default Converter;

console.log(Converter.inchesToCentimeters(12)) 

File: module/module.py
def converter():
    FOOT = 30.48
    INCH = 2.54 
    MILE = 1609.34

    class PublicClass(object):
        def inches_to_centimeters(inches):
            return inches * INCH
        def feet_to_meters(feet): 
            return feet * FOOT
        def milesToKilometers( miles):
            return miles * MILE
    return PublicClass

module_converter = converter()
print(module_converter.feet_to_meters(12))